code/datums/elements/watery_tile.dm

```diff
@@ -0,0 +1,17 @@
+/datum/element/watery_tile
+
```

Comment:
Your CI failure is happening because turfs handle signals weird, you need to give this `ELEMENT_DETACH_ON_HOST_DESTROY`.